### PR TITLE
add login time, update time when its 1 day and bonus money.

### PR DESCRIPTION
**Pull request by josia-santoso**
created a function to update login time when it has pass 1 day from the previous login time, and they will get bonus coins/money/balance.

### CODE REVIEW

File: src/screen/GameScreen.java [hunk 1]
```diff
@@ -818,4 +818,5 @@ else if (!item.getIsGet() &&
 			item.setIsGet();
 		}
 	}
+
```
Comment (fabiantran5200) [inline src/screen/GameScreen.java:821]: Delete this line

**Review by fabiantran5200 — COMMENTED** [review]
Overall good just small things

File: src/engine/Core.java [hunk 2]
```diff
@@ -247,7 +246,11 @@ public static void main(final String[] args) {
 			}
 
 		} while (returnCode != 0);
+		try {
```
[the remark this answers is not in the document]
Comment (josia-santoso) [reply src/engine/Core.java:249]: okayy

File: src/screen/GameScreen.java (hunk 1 above)
Comment (josia-santoso) [reply src/screen/GameScreen.java]: okay

**Review by fabiantran5200 — APPROVED** [review]
lgtm

**Pull request merged**